src/test/scala/mesosphere/marathon/api/validation/V2AppDefinitionValidatorTest.scala

```diff
@@ -1,5 +1,6 @@
 package mesosphere.marathon.api.validation
-
+// TODO AW: test
```

Comment:
Yes, reenabling the tests is the next step I wanted to do. Until then there should be no merge.